perf/src/sigverify.rs

```diff
@@ -141,6 +142,62 @@ fn verify_packet(packet: &mut Packet) {
         pubkey_start = pubkey_end;
         sig_start = sig_end;
     }
+
+    check_for_simple_vote_transaction(packet, msg_start).ok();
```

Comment:
We should actually put this in `do_get_packet_offsets` because `verify_packet` isn't called when using GPU to do sigverify. Looks like we made the same mistake for the tracer pubkey check but we can fix that later

Reply:
Cool, didn't know that. created issue #20018 20018

Reply:
I meant we can move the tracer check later. We should move the vote check now

Reply:
then, any objection to move tracer check all together?